<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Ignitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ignitor.qrc">
    <normaloff>:/Icons/ignitor</normaloff>:/Icons/ignitor</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2">
     <widget class="QLabel" name="labelSpeed">
      <property name="text">
       <string>Speed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBoxSpeedSet">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="labelSpeedSet">
      <property name="text">
       <string>Rpm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButtonGenerate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEditSpeedReal">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <layout class="QGridLayout" name="gridLayoutTimings">
      <item row="0" column="4">
       <widget class="QLabel" name="labelValue">
        <property name="text">
         <string>Value</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelTiming">
        <property name="text">
         <string>Timing</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelShift">
        <property name="text">
         <string>Shift</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelRpm">
        <property name="text">
         <string>Rpm</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelIndex">
        <property name="text">
         <string>Index</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>236</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGenerator">
    <property name="title">
     <string>Generator</string>
    </property>
   </widget>
   <addaction name="menuPort"/>
   <addaction name="menuGenerator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ignitor.qrc"/>
 </resources>
 <connections/>
</ui>
